Parse/src/main/java/com/parse/ParseAWSRequest.java

```diff
@@ -19,14 +19,18 @@
  * Request returns a byte array of the response and provides a callback the progress of the data
  * read from the network.
  */
-/** package */ class ParseAWSRequest extends ParseRequest<byte[]> {
+/** package */ class ParseAWSRequest extends ParseRequest<Void> {
```

Comment:
Basically, what `ParseAWSRequest` do is to read response stream and write data to a tempFile. Since we have already injected the tempFile (which means the caller already holds the file pointer), I change `byte[]` to `Void`. But if you think return the tempFile is cleaner, I can change this.